recipe/meta.yaml

```diff
@@ -8,53 +8,68 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/qpd-{{ version }}.tar.gz
-  sha256: 70e262130c71e8b4fd12c0e3b93030c9c19f949e2b1c3bd63025c43d03c6ef00
+  sha256: 2bf3fb0a4cf2307b7f3df06ac6d7a4396da7867350d17f3c37cb415fbb8089f7
 
 build:
+  # s390x is missing triad  
+  skip: true  # [py<37 or s390x]
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vvv
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
+    - setuptools
+    - wheel
   run:
     - adagio
-    - antlr4-python3-runtime >=4.10
-    - pandas >=1.0.2
-    - python >=3.6
-    - sqlalchemy
-    - triad
+    - antlr4-python3-runtime >=4.11.1,<4.12
+    - pandas >=1.2.0
+    - python
+    - triad >=0.8.0
 
 test:
-  source_files:
-    - tests
+  #source_files: 
```

Comment:
is this needed ?

Reply:
This is commented out, so when we sort out constraints with ray package, we could uncomment and switch on tests.